### DISCUSSION TITLE
2023-12-14 스터디 회의

### DISCUSSION THREAD

**Discussion by kses1010** [code-itch/effective-typescript#10] [General]
### 스터디 회의
- 발표할 때 세줄 요약 추가하기
- 발표시 자신의 중심 주제 정하기
- 바쁘면 발표보단 퀴즈

**Comment by BeMatthewsong** [comment]
요약 3줄,
자신의 토픽 중에서 핵심 주제 정하기,
너무 책을 베끼는 형식의 글은 자제하기,
해당 주제에 공유하기 좋은 자료가 있다면 첨부하기
